applications/HDF5Application/custom_python/add_custom_io_to_python.cpp

```diff
@@ -0,0 +1,90 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//
+//
+
+
+// System includes
+
+// External includes
+#include <boost/python.hpp>
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "includes/io.h"
+
+// Application includes
+#include "custom_io/hdf5_file.h"
+#include "custom_io/hdf5_file_serial.h"
+#include "custom_io/hdf5_model_part_io.h"
+#include "custom_io/hdf5_nodal_solution_step_data_io.h"
+#ifdef KRATOS_USING_MPI
+#include "custom_io/hdf5_file_parallel.h"
+#include "custom_io/hdf5_partitioned_model_part_io.h"
+#endif
+
+namespace Kratos
+{
+
+namespace Python
+{
+
+void AddCustomIOToPython()
+{
+    using namespace boost::python;
+
+    class_<HDF5::File, HDF5::File::Pointer, boost::noncopyable >("HDF5File", init<Parameters&>())
```

Comment:
I understand why you placed everything on a different namespace, but it generates some confusion with the python interface, since this one does not follow the same convention and just prepends `HDF5` to class names. What would you think of adding all these functions to an `hdf5` module in Python to replicate the same structure?

Reply:
fixed.